Intermittent failure in math test

I just had a `make test` (against 0.6.0-dev.711) spitting out
```
	From worker 7:	     * math                 Test Failed
	From worker 7:	  Expression: lgamma(-300im) ≅ -473.1718507425924 - 1410.3490664555823im
	From worker 7:	   Evaluated: -473.1718507425924 - 1410.349066455582im isequal -473.1718507425924 - 1410.3490664555823im
error in running finalizer: Base.MethodError(f=Base.#real(), args=(Base.RemoteChannel{Base.Channel{Int64}}(where=1, whence=1, id=2),))
error in running finalizer: Base.MethodError(f=Base.#real(), args=(Base.RemoteChannel{Base.Channel{Int64}}(where=1, whence=1, id=2),))
jl_uv_writecb() ERROR: bad file descriptor EBADF
jl_uv_writecb() ERROR: bad file descriptor EBADF
jl_uv_writecb() ERROR: bad file descriptor EBADF
jl_uv_writecb() ERROR: bad file descriptor EBADF
jl_uv_writecb() ERROR: bad file descriptor EBADF
```
some more pages of that line repeating, ending in
```
Exception running test math :
On worker 7:
LoadError: There was an error during testing
 in record at ./test.jl:397
 in do_test at ./test.jl:281
 in include_string at ./loading.jl:478
 in include_from_node1 at ./loading.jl:544
 in macro expansion at ./util.jl:232 [inlined]
 in runtests at /home/hol/Projects/julia/test/testdefs.jl:7
 in #619 at ./multi.jl:1425
 in run_work_thunk at ./multi.jl:1005
 in macro expansion at ./multi.jl:1425 [inlined]
 in #618 at ./event.jl:68
while loading /home/hol/Projects/julia/test/math.jl, in expression starting on line 658
ERROR: LoadError: Some tests exited with errors.
 in (::##11#18)() at /home/hol/Projects/julia/test/runtests.jl:72
 in cd(::##11#18, ::String) at ./file.jl:69
 in include_from_node1(::String) at ./loading.jl:541
 in process_options(::Base.JLOptions) at ./client.jl:262
 in _start() at ./client.jl:326
while loading /home/hol/Projects/julia/test/runtests.jl, in expression starting on line 26
Makefile:12: recipe for target 'all' failed
make[1]: *** [all] Error 1
Makefile:569: recipe for target 'test' failed
make: *** [test] Error 2
```

(All other tests completed successfully, output omitted.)

Re-running the tests, everything looks fine. Can anyone make sense of that?

--- a/test/numbers.jl
+++ b/test/numbers.jl
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+const ≣ = isequal # convenient for comparing NaNs
+
 # basic booleans
 @test true
 @test !false
@@ -54,8 +56,6 @@
 @test 2.0 * 3.0 == 6.
 @test min(1.0,1) == 1
 
-const ≅ = isequal # convenient for comparing NaNs
-
 # min, max and minmax
 @test min(1) === 1
 @test max(1) === 1
@@ -63,32 +63,32 @@ const ≅ = isequal # convenient for comparing NaNs
 @test minmax(5, 3) == (3, 5)
 @test minmax(3., 5.) == (3., 5.)
 @test minmax(5., 3.) == (3., 5.)
-@test minmax(3., NaN) ≅ (NaN, NaN)
-@test minmax(NaN, 3) ≅ (NaN, NaN)
-@test minmax(Inf, NaN) ≅ (NaN, NaN)
-@test minmax(NaN, Inf) ≅ (NaN, NaN)
-@test minmax(-Inf, NaN) ≅ (NaN, NaN)
-@test minmax(NaN, -Inf) ≅ (NaN, NaN)
-@test minmax(NaN, NaN) ≅ (NaN, NaN)
+@test minmax(3., NaN) ≣ (NaN, NaN)
+@test minmax(NaN, 3) ≣ (NaN, NaN)
+@test minmax(Inf, NaN) ≣ (NaN, NaN)
+@test minmax(NaN, Inf) ≣ (NaN, NaN)
+@test minmax(-Inf, NaN) ≣ (NaN, NaN)
+@test minmax(NaN, -Inf) ≣ (NaN, NaN)
+@test minmax(NaN, NaN) ≣ (NaN, NaN)
 @test min(-0.0,0.0) === min(0.0,-0.0)
 @test max(-0.0,0.0) === max(0.0,-0.0)
 @test minmax(-0.0,0.0) === minmax(0.0,-0.0)
 @test max(-3.2, 5.1) == max(5.1, -3.2) == 5.1
 @test min(-3.2, 5.1) == min(5.1, -3.2) == -3.2
 @test max(-3.2, Inf) == max(Inf, -3.2) == Inf
-@test max(-3.2, NaN) ≅ max(NaN, -3.2) ≅ NaN
+@test max(-3.2, NaN) ≣ max(NaN, -3.2) ≣ NaN
 @test min(5.1, Inf) == min(Inf, 5.1) == 5.1
 @test min(5.1, -Inf) == min(-Inf, 5.1) == -Inf
-@test min(5.1, NaN) ≅ min(NaN, 5.1) ≅ NaN
-@test min(5.1, -NaN) ≅ min(-NaN, 5.1) ≅ NaN
+@test min(5.1, NaN) ≣ min(NaN, 5.1) ≣ NaN
+@test min(5.1, -NaN) ≣ min(-NaN, 5.1) ≣ NaN
 @test minmax(-3.2, 5.1) == (min(-3.2, 5.1), max(-3.2, 5.1))
 @test minmax(-3.2, Inf) == (min(-3.2, Inf), max(-3.2, Inf))
-@test minmax(-3.2, NaN) ≅ (min(-3.2, NaN), max(-3.2, NaN))
-@test (max(Inf,NaN), max(-Inf,NaN), max(Inf,-NaN), max(-Inf,-NaN)) ≅ (NaN,NaN,NaN,NaN)
-@test (max(NaN,Inf), max(NaN,-Inf), max(-NaN,Inf), max(-NaN,-Inf)) ≅ (NaN,NaN,NaN,NaN)
-@test (min(Inf,NaN), min(-Inf,NaN), min(Inf,-NaN), min(-Inf,-NaN)) ≅ (NaN,NaN,NaN,NaN)
-@test (min(NaN,Inf), min(NaN,-Inf), min(-NaN,Inf), min(-NaN,-Inf)) ≅ (NaN,NaN,NaN,NaN)
-@test minmax(-Inf,NaN) ≅ (min(-Inf,NaN), max(-Inf,NaN))
+@test minmax(-3.2, NaN) ≣ (min(-3.2, NaN), max(-3.2, NaN))
+@test (max(Inf,NaN), max(-Inf,NaN), max(Inf,-NaN), max(-Inf,-NaN)) ≣ (NaN,NaN,NaN,NaN)
+@test (max(NaN,Inf), max(NaN,-Inf), max(-NaN,Inf), max(-NaN,-Inf)) ≣ (NaN,NaN,NaN,NaN)
+@test (min(Inf,NaN), min(-Inf,NaN), min(Inf,-NaN), min(-Inf,-NaN)) ≣ (NaN,NaN,NaN,NaN)
+@test (min(NaN,Inf), min(NaN,-Inf), min(-NaN,Inf), min(-NaN,-Inf)) ≣ (NaN,NaN,NaN,NaN)
+@test minmax(-Inf,NaN) ≣ (min(-Inf,NaN), max(-Inf,NaN))
 
 # fma
 let x = Int64(7)^7

